projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -105,7 +108,26 @@ module.exports = async function (...args) {
 
 	const inputPathExists = fs.existsSync(BUILD_CONFIG.input);
 
-	if (!cssOnly) {
+	const files = expandGlobs(
+		[
+			path.join(BUILD_CONFIG.input, '**/*.js'),
```

Comment:
Not sure if you need to add soy here, since the code below is building it too :thinking: